content/en/releases/_index.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: "Releases"
+menu:
+  main:
+    weight: 30
+---
+
+{{% blocks/cover title="OpenTelemetry Tracing Specification now 1.0!" height="auto" %}}
+
+Our goal is to provide a generally available, production quality release for
+the tracing data source across most OpenTelemetry components in the first half
+of 2021. Several components have already reached this milestone!
+
+We expect metrics to reach the same status in the second half of 2021 and are
+targeting logs in 2022.
+
+{{% /blocks/cover %}}
+
+{{% blocks/section type="section" color="primary" %}}
```

Comment:
Maybe this should be secondary color? The blue on blue link text is hard to read.

Reply:
Yeah, we had the same issue on https://opentelemetry.io/about/. I opened #384 to track this and should have linked here.

Given #383 and #303 I actually have a PR on top of this one that switches to the layout that community/registry use while adding a dark middle section so links are visible.

Reply:
See #386 -- perhaps we could merge this then land the supplemental PR. @austinlparker thoughts?

Reply:
yeah, that works for me. #386 looks solid.